main.ledger

```diff
@@ -1365,6 +1375,11 @@
     Assets:Bank:Checking
     ; Receipt: b0ec530c27a10049e56945b6b57e6ed7.pdf
 
+2015/10/09 Someone didn't cash their check
```

Comment:
We pay Wells Fargo from Max Wofford's salary account. Just leave a comment
saying he didn't deposit his check.

On Mon, Apr 11, 2016 at 11:38 PM, JESSICA <notifications@github.com> wrote:

> In main.ledger
> <https://github.com/hackclub/finances/pull/12#discussion_r59326972>:
>
> > @@ -1365,6 +1375,11 @@
> >      Assets:Bank:Checking
> >      ; Receipt: b0ec530c27a10049e56945b6b57e6ed7.pdf
> >
> > +2015/10/09 Someone didn't cash their check
>
> Nor this
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/finances/pull/12/files/e26cdeb95d7463d6e787d9ade18c2269ca5489a3#r59326972>
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>